### PR TITLE
Debounce trk.pm-srv.co urls

### PR DESCRIPTION
**Pull request by iambrianfung**
example link:
https://trk.pm-srv.co/v2/trk?adid=sKdvGxKy&akey=1e0a2692-9008-4626-992e-e8e64d86ee09&rurl=https%3A%2F%2Fgetcardealstoday.com%2Fdeals%3Futm_source%3DBrave%26utm_campaign%3D%26adv%3DSubaru%26oem%3DSubaru%26make%3DSubaru%26model%3DCrosstrek%26year%3D2023%26PID%3D8CU6NBY75%26device%3D%7Bdevice%7D%26OS%3D%7BOS%7D%26keyword%3D%7Bkeyword%7D

### CODE REVIEW

File: brave-lists/debounce.json [hunk 1]
```diff
@@ -47,6 +47,15 @@
     "action": "redirect",
     "param": "Url"
   },
+  {
+    "include": [
+      "*://trk.pm-srv.co/*"
```
Comment (fmarier) [inline brave-lists/debounce.json:52]: I would make this a bit tighter to ensure we only debounce the links we know:
```
"*://trk.pm-srv.co/v2/trk?*"
```

**Review by fmarier — CHANGES_REQUESTED** [review]
The sample URL returns a 404 for me. I guess that ad is no longer active?

File: brave-lists/debounce.json (hunk 1 above)
Comment (fmarier) [reply brave-lists/debounce.json:52]: For example, if there's a `/v3/` later, it may not use the `rurl` param the same way.

Comment (iambrianfung) [reply brave-lists/debounce.json]: sound sensible

**Comment by fmarier** [comment]
Oops, I meant to approve before merging :)

Anyways, thanks @iambrianfung !

**Pull request merged**